File: 프로그래머스 Lv.1/행렬의 덧셈.py
def solution(arr1, arr2):     
    a=len(arr1)
    b=len(arr1[0])
    answer = [[] for _ in range (a)]      
    for i in range (a):
        for j in range(b):
            answer[i].append(arr1[i][j]+arr2[i][j])       
    return answer

File: 코틀린/행렬의 덧셈.kt
class Solution {
    fun solution(arr1: Array<IntArray>, arr2: Array<IntArray>): Array<IntArray> {
        var answer = arrayOf<IntArray>()
        var s=arr1[0].size
        
        for (i in 0..arr1.size-1){
            var array=IntArray(s)
            for (j in 0..s-1){
                array[j]=arr1[i][j]+arr2[i][j]
            }
            answer=answer.plus(array)
        }
                
        return answer
    }
}
